webpack.dev.js

```diff
@@ -30,8 +30,11 @@ module.exports = merge(common({ mode }), {
     },
     compress: false,
     headers: {
+      // https://github.com/codemirror/codemirror5/issues/6707
+      // "default-src 'none';
+      // style-src blob: 'nonce-tkn-dev';
       'Content-Security-Policy':
-        "default-src 'none'; img-src 'self'; script-src 'self' 'unsafe-eval'; style-src blob: 'nonce-tkn-dev'; connect-src 'self' wss: ws:; font-src 'self' https://1.www.s81c.com;"
+        "img-src 'self'; script-src 'self' 'unsafe-eval' 'unsafe-inline'; connect-src 'self' wss: ws:; font-src 'self' https://1.www.s81c.com;"
```

Comment:
Can we keep `default-src 'none'` and just add the required values for `style-src`? Maybe `style-src 'self' 'unsafe-inline';`?

It would also need to be updated for the production build in https://github.com/tektoncd/dashboard/blob/2b3b243148e2d6829fb749024e987a2f8b17c025/pkg/router/router.go#L49